tests/unit/h/views/account_signup_test.py

```diff
@@ -169,7 +168,7 @@ def pyramid_request(self, pyramid_request):
     def views(self, pyramid_request):
         return SignupViews(sentinel.context, pyramid_request)
 
-    @pytest.fixture
+    @pytest.fixture(autouse=True)
```

Comment:
May want to add `@pytest.mark.usefixture("frozen_time")` to the tests in question rather than adding `autouse=True` here: applying this fixture to every single unittest may slow down the tests.

Reply:
Done